src/index.ts

```diff
@@ -650,6 +650,7 @@ export class ExtensibilitySdk {
     outreachContext.locale = runtime.locale;
     outreachContext.theme = runtime.theme;
     outreachContext.userIdentifier = runtime.userIdentifier;
+    outreachContext.config = runtime.configuration;
```

Comment:
what does runtime configuration contain? It could be some private keys. outreachContext.config should only contain values that are set in the app configuration in the dev portal.

Furthermore, `config` is an object and thus this will get assigned by reference. Where as locale, theme and userIdenfier are scalar and the value will be copied. We should definitely copy the object here.

Reply:
this is how it'd look like. I've set some values in app configuration in devportal, than filled out some values when installing an app:

<img width="545" alt="image" src="https://github.com/getoutreach/extensibility-sdk/assets/22118704/7eb142ad-0810-45bc-b8e8-e666a8df31ab">